contracts/cw20/NOTICE

```diff
@@ -0,0 +1,14 @@
+cw20
+Copyright 2021,2022 Confio GmbH
+
```

Comment:
Please add our copyright in this line.

Reply:
Why do only cw20 and cw721 need our copyright? We can abandon rights because of templates.